src/html/snis/index.html

```diff
@@ -0,0 +1,62 @@
+<div class="container" infinite-scroll="loadMore()" infinite-scroll-distance="5" infinite-scroll-use-document-bottom="true">
+    <a href="#/snis/add" class="btn-floating btn-large waves-effect waves-light right" ng-show="snis.length >= 1">
+        <i class="material-icons">add</i>
+    </a>
+
+    <h3 class="header">snis</h3>
+
+    <p class="flow-text center" ng-show="total === null" style="margin-top:50px;">
+        <app-loader></app-loader>
+    </p>
+
+    <div class ng-show="total === 0">
+        <p class="flow-text center">
+            You haven't created any SNI in Kong yet.
+        </p>
+        <p class="center">
+            <a href="#/snis/add" class="waves-effect waves-light btn">
+                <i class="material-icons left">add_box</i>
+                Add Sni
+            </a>
+        </p>
+    </div>
+
+    <table class="bordered" ng-show="total > 0">
+        <thead>
+        <tr>
+            <th>Name</th>
+            <th>Created</th>
+            <th></th>
+        </tr>
+        </thead>
+
+        <tbody>
+        <tr ng-repeat="sni in snis">
+            <td>{{sni.name}}</td>
+            <td>{{sni.created_at | date}}</td>
+            <td class="right">
+                <a class="btn-floating waves-effect waves-light" href="#/snis/{{sni.id}}">
```

Comment:
After creating an SNI, clicking this link would bring me back to `/snis`. So I'm unable to edit an SNI.